fedimintd/src/ui.rs

```diff
@@ -236,6 +241,13 @@ async fn params_page(
     UrlConnection {
         ro_bitcoin_rpc_type,
         ro_bitcoin_rpc_url,
+        bind_p2p: match env::var("FM_BIND_P2P") {
+            Ok(s) => Some(s),
+            _ => None,
+        },
+        p2p_url: env::var("FM_P2P_URL").unwrap_or("".to_string()),
```

Comment:
Seems unsafe to default to an empty string, better to return an error.

Reply:
This just pre-fills a value in the HTML form based on an environment variable. If no environment variable is set, then the form field is empty and user needs to fill it in themselves.

Reply:
Could be nice if we auto-generated the bind addresses.

Reply:
What do you mean auto-generated? Choose a random open port?

Reply:
Shall I try adding auto-generation as part of this PR or should that be done separately?

Reply:
Problem with that is that this port needs to be opened or forwarded somehow. And I think this port is also used for DKG, right? So unless the user had basically all their ports open or opened the randomly selected port after submitting this form, then DKG wouldn't complete?